packages/tsconfig-reference/copy/en/options/allowUnreachableCode.md

```diff
@@ -9,7 +9,7 @@ When:
 - `true` unreachable code is ignored
 - `false` raises compiler errors about unreachable code
 
-These warnings are only about code which is provably unreachable due to the use of JavaScript syntax, for example:
+These warnings are only about code which is probably unreachable due to the use of JavaScript syntax, for example:
```

Comment:
This isn't a typo, it means you can prove it is unreachable - the context switch to probably makes it seem unreliable instead :D

Reply:
Oh! Sorry, I thought it was a typo. Me and my small vocab 😢 

I'll reverse it!